rsconf/component/base_os.py

```diff
@@ -37,6 +37,7 @@ def internal_build_compile(self):
     def internal_build_write(self):
         jc = self.j2_ctx
         z = jc.base_os
+        z.has_component_postfix = self.buildt.has_component("postfix")
```

Comment:
Should this be in compile?

Reply:
It can't be. This the pass1, pass2 problem. postfix hasn't executed yet so it has to be in write.

Reply:
5b04816